From 6af2f2785903886c2371aefe43d1d60dcc3acae8 Mon Sep 17 00:00:00 2001
From: Andre McCurdy <armccurdy@gmail.com>
Date: Mon, 30 Nov 2015 14:45:56 -0800
Subject: [PATCH] link BCM Nexus backend with nxclient

The BCM Nexus backend depends on nxclient (see commit below) so should
link with libnxclient.so. That sometimes happens "by chance" due to
libs listed by egl.pc and/or glesv2.pc, but making it explicit is
clearer.

  https://github.com/Metrological/WebKitForWayland/commit/5e8808a238e4dc919b133d485dd2b57cf1205b67

Signed-off-by: Andre McCurdy <armccurdy@gmail.com>
---
 Source/WPE/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WPE/CMakeLists.txt b/Source/WPE/CMakeLists.txt
index 5526d40..756f0d3 100644
--- a/Source/WPE/CMakeLists.txt
+++ b/Source/WPE/CMakeLists.txt
@@ -92,6 +92,7 @@ if (USE_WPE_BACKEND_BCM_NEXUS)
 
         Source/ViewBackend/BCMNexus/ViewBackendBCMNexus.cpp
     )
+    list(APPEND WPE_LIBRARIES nxclient)
 endif ()
 
 if (USE_WPE_BACKEND_BCM_RPI)
-- 
1.9.1

